plugins/rtmp-services/data/services.json

```diff
@@ -1103,55 +1103,21 @@
         {
             "name": "Switchboard Live",
             "alt_names": [
-                "Switchboard Live (Joicaster)"
+                "Switchboard Live"
```

Comment:
I'm pretty sure this will break user configurations if they still have the old Service dropdown entry selected. `alt_names` is used to leave the Service selected for the user, and fetching the `servers` list attached.

Reply:
For old selected URLs, we'll create CNAMEs. I've also reverted the `alt_names` change

Reply:
We're also informing our OBS users about this update to make necessary changes

Reply:
This isn't about your service, but how it appears in OBS. You can change the name, you just need the old one to be listed in `alt_names`, which is invisible to the user and only used internally for anyone with an old config to know what service is selected.

Reply:
Understood. I reverted the changes to the `alt_names`